### PR TITLE
first commit new master

### PR DESCRIPTION
**Pull request by dcuoliveira**

### CODE REVIEW

File: hawkes/simulate_smalltick.py
Comment (konqr) [inline hawkes/simulate_smalltick.py:1]: @dcuoliveira can you pls undelete this file or move it to backup/hawkes?

File: hawkes/simulate_optimized.py
Comment (konqr) [inline hawkes/simulate_optimized.py:1]: @dcuoliveira can you pls undelete this file or move it to backup/hawkes?

File: hawkes/experiments/EDA.ipynb
Comment (konqr) [inline hawkes/experiments/EDA.ipynb:1]: @dcuoliveira can you pls undelete this file or move it to backup/hawkes/experiments? same applies to all of the files in hawkes/experiments

File: Benchmarking.ipynb
Comment (konqr) [inline Benchmarking.ipynb:1]: @dcuoliveira can you pls undelete this file or move it to backup/hawkes/experiments?

File: hawkes/CreateLOB.py
Comment (konqr) [inline hawkes/CreateLOB.py:1]: @dcuoliveira can you pls undelete this file or move it to backup/hawkes? this is an optimized version of the createLOB function

File: runner.ipynb
Comment (konqr) [inline runner.ipynb:1]: @dcuoliveira can you pls undelete this file or move it to backup/hawkes/experiments?

File: simulated_data/fake_0_0_19_inputRes
Comment (konqr) [inline simulated_data/fake_0_0_19_inputRes:1]: @dcuoliveira can you pls undelete this file or move it to backup/hawkes/simulated_data? same for all files under hawkes/simulated_data

File: src/pricePathsConditionalEvent.ipynb
Comment (konqr) [inline src/pricePathsConditionalEvent.ipynb:1]: @dcuoliveira can you pls **delete** this file or move it to backup/hawkes/experiments?

File: src/runner.ipynb
Comment (konqr) [inline src/runner.ipynb:1]: @dcuoliveira can you pls delete this file or move it to hawkes/drafts?

File: src/sif_exponential_kernel_mle.ipynb
Comment (konqr) [inline src/sif_exponential_kernel_mle.ipynb:1]: @dcuoliveira can you pls delete this file or move it to hawkes/drafts?

File: src/uncertaintyZones.ipynb
Comment (konqr) [inline src/uncertaintyZones.ipynb:1]: @dcuoliveira can you pls delete this file or move it to backup/hawkes/experiments?

**Review by konqr — CHANGES_REQUESTED** [review]
just need undeleting and moving around of files. rest looks good.

**Comment by dcuoliveira** [comment]
Added missing files and refactored scripts to drafts/backup

**Review by konqr — APPROVED** [review]

**Pull request merged**